_rules/menuitem-non-empty-name-m6b1q3.md

```diff
@@ -0,0 +1,153 @@
+---
+id: m6b1q3
+name: Menuitem has non-empty accessible name
+rule_type: atomic
+description: |
+  This rule checks that each element with a `menuitem` role has a non-empty accessible name.
+accessibility_requirements:
+  wcag20:4.1.2: # Name, Role, Value (A)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+input_aspects:
+  - Accessibility Tree
+  - DOM Tree
+  - CSS Styling
+acknowledgments:
+  authors:
+    - Wilco Fiers
+---
+
+## Applicability
+
+This rule applies to HTML elements with a `menuitem` [semantic role][] that are [included in the accessibility tree][].
```

Comment:
We seem to have various ways of saying that in the various "has accessible name" rules 🙈 
So, this is OK but we might want to unify all of them a bit at a later point.

Reply:
I intentionally kept this one a little simpler than the others because there are fewer conditions to consider here.